backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/predicate/dsl/impl/ElasticsearchKnnPredicateFieldStep.java

```diff
@@ -9,16 +9,16 @@
 import org.hibernate.search.backend.elasticsearch.search.predicate.dsl.ElasticsearchKnnPredicateOptionsStep;
 import org.hibernate.search.backend.elasticsearch.search.predicate.spi.ElasticsearchKnnPredicateBuilder;
 import org.hibernate.search.engine.search.predicate.dsl.SearchPredicateFactory;
-import org.hibernate.search.engine.search.predicate.dsl.impl.AbstractKnnPredicateFieldStep;
+import org.hibernate.search.engine.search.predicate.dsl.spi.AbstractKnnPredicateFieldStep;
 import org.hibernate.search.engine.search.predicate.dsl.spi.SearchPredicateDslContext;
 import org.hibernate.search.engine.search.predicate.spi.PredicateTypeKeys;
 
-public final class ElasticsearchKnnPredicateFieldStepImpl
+public final class ElasticsearchKnnPredicateFieldStep
```

Comment:
this seems to make sense, but I didn't want it to sneak in so I've made these changes in a temporary commit so you can see them isolated 😄

Reply:
If jqassistant is happy, I'm happy: feel free to squash this.